cmd/minishift/cmd/config/view.go

```diff
@@ -74,7 +75,9 @@ func configView() error {
 		if excluded {
 			continue
 		}
-
+		if configViewFormat != DefaultConfigViewFormat {
+			configViewFormat = fmt.Sprintln(configViewFormat)
```

Comment:
@praveenkumar You are just appending a newline with the help of Sprintln isn't it?